src/streamArrayParser.ts

```diff
@@ -60,9 +60,10 @@ export class StreamArrayParser extends Transform {
     this._level = 0;
     this.rpc = rpc;
     this.cancelController = hasAbortController()
-      ? // eslint-disable-next-line no-undef
-        new AbortController()
+      ? new AbortController()
       : new NodeAbortController();
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Same here. I understand it might be a problem with the new TypeScript? Let's figure out what's going on.

Reply:
Updated.